doc/register.md

```diff
@@ -5,23 +5,43 @@
 
 @signature `route.register(rule [, defaults])`
 
-Create a url matching rule. Optionally provide defaults that will be applied to the underlying [can-route.data] when the rule matches.
-
-The following sets `route.data.page = "cart"` when the url is `#cart` and
-`route.data.page = "home"` when the url is `#`.
-
-```js
-route.register( "{page}", { page: "home" } );
-```
-
-@param {String} rule the fragment identifier to match.  The fragment identifier should contain characters (a-Z), optionally wrapped in braces ( { } ). Identifiers wrapped in braces are interpreted as being properties on can-route’s map. Examples:
-
-```js
-route.register( "{foo}" );
-route.register( "foo/{bar}" );
-```
-
-@param {Object} [defaults] An object of default values. These defaults are applied to can-route’s map when the route is matched.
-
-@return {Object} The internal route object.  Use values on this object with caution. It is
- subject to change.
+  Create a url matching rule. Optionally provide defaults that will be applied to the underlying [can-route.data] when the rule matches.
+
+  The following sets `route.data.page = "cart"` when the url is `#cart` and `route.data.page = "home"` when the url is `#`.
+
+  ```html
+  <mock-url></mock-url>
+  <script type="module">
+  import "//unpkg.com/mock-url@^5.0.0";
+  import {route} from "can";
+
+  route.register( "{page}", {page: "home"} );
+  
+  route.start();
+  console.log( route.data.page ); // -> "home"
+  route.data.page = "cart";
+  </script>
+  ```
+  @codepen
+
+  @param {String} rule the fragment identifier to match.  The fragment identifier should contain characters (a-Z), optionally wrapped in braces `( { } )`. Identifiers wrapped in braces are interpreted as being properties on can-route’s map. Examples:
+
+   ```html
+   <mock-url></mock-url>
+   <script type="module">
+   import "//unpkg.com/mock-url@^5.0.0";
+   import {route} from "can";
+  
+   route.register( "{foo}" );
+   route.register( "foo/{bar}" );
+   console.log( route.data ); //-> {foo: undefined, bar: undefined}
+   
+   route.start();
+   route.data.bar = "fie"; // Url hash changes to #!foo/fie
+   </script>
+   ```
+   @codepen
+
+  @param {Object} [defaults] An object of default values. These defaults are applied to can-route’s map when the route is matched.
+
+  @return {Object} The internal route object.  Use values on this object with caution. It is subject to change.
```

Comment:
We don't show using values on the route object anymore.  We should remove this warning.

What we should show is that you can chain register calls:

```js
route.register("todos/{todoId}")
  .register("users/{userId}")
```

Reply:
Added as comment below `@return`. I didn't feel this was meaty enough to include a `## Use` section.